src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter((person) => person.sex === 'm'
+    && ((!century)
+      ? true
+      : Math.ceil(person.died / 100) === century)
+  );
```

Comment:
```suggestion
const men = people.filter(person =>
    !century
      ? (person.sex === 'm')
      : Math.ceil(person.died / 100) === century && person.sex === 'm'
  );
```

If you want to use a ternary operator and only one 'filter; then you should use it like this. Why would we need `? true` here? It's really hard to read and understand.
Use 'men' instead of 'man' (plural form man-men, woman-women)